code/__HELPERS/logging.dm

```diff
@@ -118,15 +118,26 @@
  * @param amount The amount of units transferred
  * @param reagent_names List of reagent names to log
  */
-/proc/log_reagents(var/mob/user, var/source, var/target, var/amount, var/list/reagent_names)
-	if (amount == 0)
+/proc/log_reagent_transfer(var/mob/user, var/atom/source, var/atom/target, var/amount_desired)
```

Comment:
I get that, but DM has no access modifiers, so every proc acts as an involuntary API and trans_to is going to be people's first choice, especially since this isn't on the reagents object.